domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/InvalidAgencyIdNotice.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.notice.error;
+
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.NoticeExporter;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.ErrorNotice;
+
+import java.io.IOException;
+
+public class InvalidAgencyIdNotice extends ErrorNotice {
+    private String fieldName;
+
+    public InvalidAgencyIdNotice(final String entityId) {
+        super("agency.txt", E_031,
+                "Invalid blank string",
+                "Field `agency_id` of file `agency.txt` is blank",
+                entityId);
```

Comment:
What is the difference with Missing `agency_id`?

Reply:
`MissingAgencyId` refers to the fact that field `agency_id` is null, in a `agency.txt` file containing more than one record.

`InvalidAgencyId` refers to the fact that `agency_id` is provided but blank, for example: 
`"   "`